### PR TITLE
Add top50 Pokemon and top trainer to pokemon.page

### PR DESCRIPTION
**Pull request by pki0**
## Description
- Add tables to Pokemon page
- List best IV Pokemon
- List trainer with the best Pokemon

## To Do:
- Option to hide lists and collaps? Or switch between the to lists like on heatmap/livemap? 
(This will break direct sorting of the table if you hide one of them)
- <s>support different language move files: (moves.de.json, moves.en.json)</s>
## To Fix:
- <s>Causes `PHP message: PHP Warning:  Creating default object from empty value in ../core/process/data.loader.php on line 257` -> `$move->$move_id->name = $move_name->name;`</s>

## Screenshots
relaxi
relaxi2

## Types of changes

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by pki0** [comment]
Scrutinizer suxx...

pokemon.page.php
L1: Other tag is not working here, have to be` <? `  
L3: Don't know whats wrong
Spaces: What the hell... there are spaces

**Comment by Obihoernchen** [comment]
Don't think to much about Scrutinizer I will tame the :dog:

**Comment by pki0** [comment]
Should support EN and DE now. Please review @Obihoernchen

**Comment by Obihoernchen** [comment]
~~I get 404 when sorting.
probably because of subdirectory: https://vserver.obihoernchen.net/pokemon_dev/pokemon/149~~

But it looks awesome!

Maybe change it to TOP10 + loading more like trainer page?
~~Please change `<div class="row" id="family">` to ` <div class="row area" id="family">`~~

See my PR

File: pages/pokemon.page.php [hunk 1]
```diff
@@ -286,3 +287,84 @@
...
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=individual_attack<?php echo $top_order == 'individual_attack' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#top100">A <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=individual_defense<?php echo $top_order == 'individual_defense' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#top100">D <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=individual_stamina<?php echo $top_order == 'individual_stamina' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#top100">Sta <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=move_1<?php echo $top_order == 'move_1' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#top100">1.Move <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=move_2<?php echo $top_order == 'move_2' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#top100">2.Move <i class="fa fa-sort" aria-hidden="true"></i></a></th>
```
Comment (Obihoernchen) [inline pages/pokemon.page.php]: Space pls: `1. Move`

File: pages/pokemon.page.php [hunk 2]
```diff
@@ -286,3 +287,84 @@
...
+		<table class="table">
+			<thead>
+				<tr>
+					<th>#</th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=trainer_name<?php echo $best_order == 'trainer_name' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#trainer">Name <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=cp<?php echo $best_order == 'cp' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#trainer">CP <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=IV<?php echo $best_order == 'IV' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#trainer">IV <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=move_1<?php echo $best_order == 'move_1' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#trainer">1.Move <i class="fa fa-sort" aria-hidden="true"></i></a></th>
+					<th><a href="/pokemon/<?= $pokemon->id ?>?order=move_2<?php echo $best_order == 'move_2' && !isset($_GET['direction']) ? '&direction=desc' : ''; ?>#trainer">2.Move <i class="fa fa-sort" aria-hidden="true"></i></a></th>
```
Comment (Obihoernchen) [inline pages/pokemon.page.php]: Space pls: `1. Move`

**Comment by Obihoernchen** [comment]
See: https://github.com/klatterhans/Worldopole/pull/13
And Scrutinizer shouldn't report the error anymore.

**Comment by tobias86aa** [comment]
When is this going to be merge?

**Comment by pki0** [comment]
I'll change it to Top50 now.
@Obihoernchen Top10 and loading more needs some JS skills that I don't have :)

**Comment by Obihoernchen** [comment]
I'll merge this for now and we will summon @PoUpA for the dynamic JS loading part ;) Maybe I'll take a look at it as well but I'm a little busy right now.

**Review by Obihoernchen — APPROVED** [review]

**Pull request merged**